<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferencias</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:rgb(76, 118, 82);
background-color:rgb(216, 230, 225);</string>
  </property>
  <layout class="QVBoxLayout" name="settingsLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="styleSheet">
      <string notr="true">font: bold;</string>
     </property>
     <property name="text">
      <string>Preferencias</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">color: rgb(11, 35, 21);</string>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="appearance">
      <attribute name="title">
       <string>Apariencia</string>
      </attribute>
      <layout class="QGridLayout" name="appearanceLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: bold;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Tamaño de Letra</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QSlider" name="fontSize">
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>25</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>12</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
         <property name="tracking">
          <bool>true</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>true</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font: bold;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Fuente</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QFontComboBox" name="fontComboBox"/>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: bold;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Tema </string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QComboBox" name="theme"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="security">
      <attribute name="title">
       <string>Seguridad</string>
      </attribute>
      <layout class="QGridLayout" name="securityLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="savePassword">
         <property name="styleSheet">
          <string notr="true">font: bold;
color: rgb(0, 0, 0);
</string>
         </property>
         <property name="text">
          <string>¿Guardar  contraseña en gestor de de contraseñas?</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">font: bold;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Número de intentos antes de error *</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>*Cuando existen problemas de red</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">font: bold;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Guardar documentos firmados en</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="8" column="1">
        <widget class="QPushButton" name="selectPathSaveDocs">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QSpinBox" name="requestsBeforeError">
         <property name="suffix">
          <string> intentos</string>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QLineEdit" name="saveDocsPath">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="module">
      <attribute name="title">
       <string>Módulo</string>
      </attribute>
      <layout class="QGridLayout" name="moduleLayout">
       <item row="2" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="selectModulePath">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>* Corresponde al lugar en la máquina dónde se ubica el módulo para el dispositivo de firma (PKCS11)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
font: bold;</string>
         </property>
         <property name="text">
          <string>Ubicación del Módulo *</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLineEdit" name="modulePath">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Normal</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>label_7</zorder>
      <zorder>selectModulePath</zorder>
      <zorder>label_8</zorder>
      <zorder>modulePath</zorder>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="buttonsWidget" native="true">
     <layout class="QHBoxLayout" name="buttonsWidgetLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="apply">
        <property name="styleSheet">
         <string notr="true">color: rgb(11, 35, 21);
background-color: rgb(229, 229, 229);</string>
        </property>
        <property name="text">
         <string>Aplicar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel">
        <property name="styleSheet">
         <string notr="true">color: rgb(11, 35, 21);
background-color: rgb(229, 229, 229);</string>
        </property>
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ok">
        <property name="styleSheet">
         <string notr="true">color: rgb(11, 35, 21);
background-color: rgb(229, 229, 229);</string>
        </property>
        <property name="text">
         <string>Aceptar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
